site/docs/hosting/cloudflare-workers-deno.md

```diff
@@ -0,0 +1,124 @@
+# Hosting: Cloudflare Workers (Deno)
+
+[Cloudflare Workers](https://workers.cloudflare.com/) is a public serverless computing platform that offers a convenient and simple solution for running small workloads at the [edge](https://en.wikipedia.org/wiki/Edge_computing).
+
+This guide will take you through the process of hosting your Telegram bot on Cloudflare Workers.
+
```

Comment:
Don't forget to apply the same changes to the [other doc](https://grammy.dev/hosting/cloudflare-workers.html) as well.
```suggestion

:::tip Looking for the Node.js version?
This tutorial explains how to deploy a Telegram bot to Cloudflare Workers using Deno.
If you're looking for the Node.js version, please check out [this tutorial](./cloudflare-workers.md) instead.
:::

```

Reply:
Done.